modules/tinymce/src/core/test/ts/browser/lookup/UserLookupTest.ts

```diff
@@ -76,6 +84,15 @@ describe('browser.tinymce.core.UserLookupTest', () => {
         editor.userLookup.userId = 'different-user';
       }).to.throw();
     });
+
+    it('TINY-11974: Should default to "anonymous" when no user_id is provided', () => {
+      const editor = hook.editor();
+      editor.options.unset('user_id');
+
+      editor.userLookup = createUserLookup(editor);
+
+      expect(editor.userLookup.userId).to.equal('Anonymous', 'Should default to anonymous');
```

Comment:
I think all other tests we avoid the BDD asserts. Is it because chai promises assets only has this syntax or something else.
Personally I really dislike that pattern trying to do English in some weird DSL is never good. Brings me back to the last century when I had to program a strange language called Lingo that was all English. `set the thing of something to 4` that language later switched to proper syntax `something.thing = 4`. 😃 
I don't think we need to change this now but it's inconsistent with other tests.
They are all `assert.equal(editor.userLookup.userId, 'Anonymous', 'Should default to anonymous')` style.